CVLByExample/Strings/String.spec

```diff
@@ -0,0 +1,9 @@
+rule cvlStringEqComparisonToEVMType {
+	env e;
+	string r1;
+
+	string r2 = "Blurp";
+	r1 = getBlurp(e);
+
+	assert r1 == r2, "Should succeed";
+}
```

Comment:
Maybe we can add here all the cases Johannes showed in his PR?

Reply:
It has a lot of small not interesting cases, I'm not sure if it will demonstrate the example better or confuse the reader, that's why I preferred to keep it small and clean.

Reply:
basically we support == and !=, not substring right? if so this is enough

Reply:
substring not supported yet